tmt/utils.py

```diff
@@ -735,6 +736,24 @@ def context_to_dict(context):
         for key, value in environment_to_dict(context).items()}
 
 
+def dict_to_shell(data, width=None, sort=False):
+    if not data:
+        return "\n"
+    output = ""
+    # List available attributes
+    for key, value in sorted(data.items()):
+        output += "{0}: ".format(fmf.utils.color(key, 'green'))
```

Comment:
Do we need colored output for shell?

Reply:
Dunno. This is basically stolen from fmf codebase. But I can remove it.